hub/util/generate_id.py

```diff
@@ -0,0 +1,7 @@
+from uuid import uuid4
+import numpy as np
+
+
+def generate_id(dtype: type):
```

Comment:
nice separating them, since later we might need to generate this based on data input (optionally)